docs/form.md

```diff
@@ -0,0 +1,146 @@
+  ### Properties <a name="PsFormEventProperties"></a>
+  | Name                           | Description
+  | ------------------------------ | -------------
+  | `defaultPrevented: boolean`    | `true`, if the default load success handling is prevented.
+
+  ### Functions <a name="PsFormEventFunctions"></a>
+  | Name                           | Description
+  | ------------------------------ | -------------
+  | `preventDefault(): void`       | Prevents the default load success handling from beeing executed.
+
+---
+
+  ## PsFormLoadSuccessEvent (extends PsFormEvent)  <a name="PsFormLoadSuccessEvent"></a>
+  ### Properties <a name="PsFormLoadSuccessEventProperties"></a>
+  | Name                           | Description
+  | ------------------------------ | -------------
+  | `value: any`                   | The loaded value from `loadFnc`.
+
+---
+
+  ## PsFormLoadErrorEvent (extends PsFormEvent)  <a name="PsFormLoadErrorEvent"></a>
+  ### Properties <a name="PsFormLoadErrorEventProperties"></a>
+  | Name                           | Description
+  | ------------------------------ | -------------
+  | `(readonly) error: any`        | The error object from `loadFnc`.
+
+---
+
+  ## PsFormSaveSuccessEvent (extends PsFormEvent)  <a name="PsFormSaveSuccessEvent"></a>
+  ### Properties <a name="PsFormSaveSuccessEventProperties"></a>
+  | Name                           | Description
+  | ------------------------------ | -------------
+  | `(readonly) value: any`        | The data object that was saved.
+  | `(readonly) saveResponse: any` | The saveResponse from your save functionality.
+  | `(readonly) close: boolean`    | `true`, if the "Save & close"-Button was clicked.
+
+---
+
+  ## PsFormSaveErrorEvent (extends PsFormEvent)  <a name="PsFormSaveErrorEvent"></a>
+  ### Properties <a name="PsFormSaveErrorEventProperties"></a>
+  | Name                           | Description
+  | ------------------------------ | -------------
+  | `(readonly) value: any`        | The data object that was saved.
+  | `(readonly) error: any`        | the error object from `saveFnc`.
+
+---
+
+  ## PsFormCancelEvent (extends PsFormEvent)  <a name="PsFormCancelEvent"></a>
+  
+---
+
+  ## Prerequisites/Requirements <a name="PsFormRequirements"></a>
+  
+  1. You have to override `BasePsFormService` and implement the following two functions:
+  * > `getLabel(formControl: any): Observable<string>` which should return the FormControls label.
+  * > `mapDataToError(errorData: IPsFormErrorData[]): Observable<IPsFormError[]>` which should return `IPsFormError` with the needed information `errorText` and `data`.
+
+  2. Import the PsFormBaseModule forRoot with the created service in your AppModule. Like this:
+  `PsFormBaseModule.forRoot(DemoPsFormsService)`
+
+---
+
+  ## Implementation <a name="PsFormImplementation"></a>
+  Import the module into your module. 
+
+  ```ts | js
+    @NgModule({
+      declarations: [MyComponent],
+      imports: [PsFormModule]

```

Comment:
Should we show how to register a implementation for BasePsFormService here?

Reply:
As mentioned from you we should do this in a "core.md" or something. I can provide such file in a following PR